src/main/java/net/mcreator/ui/minecraft/SoundSelector.java

```diff
@@ -47,8 +48,9 @@ public SoundSelector(MCreator frame) {
 			Arrays.sort(sounds);
 
 			String s = (String) JOptionPane
-					.showInputDialog(frame, "Please select sound:", "Selection", JOptionPane.PLAIN_MESSAGE, null,
-							sounds, sounds[0]);
+					.showInputDialog(frame, L10N.t("dialog.list_field.sound_message"),

```

Comment:
One more I found, this one is not a list field, so change that part

Reply:
Done.